config/cable.yml

```diff
@@ -2,8 +2,9 @@ development:
   adapter: async
 
 test:
-  adapter: async
+  adapter: test
 
 production:
   adapter: redis
-  url: redis://localhost:6379/1
+  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
+  channel_prefix: link_checker_api_production
```

Comment:
Was this change automatic? Quite surprised to see the application name in the default output!

Reply:
Yes that was automatic